### PR TITLE
[Addition] Popular alternatives to OpenAI, Github Copilot and Chat GPT

### PR DESCRIPTION
**Pull request by vng3nc3**
# Description 

1. These are alternatives to the famous LLM related apps being used at the moment
2. Publicly these companies have not shown direct support for oppression , Making them ideal at the current moment in time
3. I have personally used them and they are as good as the incumbents
4. This list will expand based on my research , I will add those which I personally use

### CODE REVIEW

File: dataset/companies/AI.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+- Name: "OpenAI"
+  Description: "AI Research"
+  Website: "https://openai.com/"
+  Alternatives:
+    [
```
Comment (abarmawi) [inline dataset/companies/AI.yaml:5]: Data should be in `yaml` format. Please run `npm run validate` before the commit.

**Comment by hosamaly** [comment]
Could you please share some evidence about these companies' support for Israel?

**Comment by 0x0OZ** [comment]
> Could you please share some evidence about these companies' support for Israel?

@hosamaly Here is one

https://www.linkedin.com/posts/hamzah-hajeir_more-no-mercy-idf-dont-stop-tal-broda-activity-7141392967751127041-oopO?utm_source=share&utm_medium=member_android

File: dataset/companies/AI.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+- Name: "HuggingFace"
```
Comment (abarmawi) [inline dataset/companies/AI.yaml:1]: Are these Israeli tech of alternatives?
If they are Israeli tech , then i can approve, else then we need add the Israeli tech and their alternatives

example

```
- Name: Israeli tech name
   Website: ...
   Alternatives:
     - Name: HuggingFace
        Website: ...
```

**Comment by abarmawi** [comment]
Closing for due to no activity

**Pull request closed without merging**